src/lcm/example_models.py

```diff
@@ -125,3 +129,29 @@ def absorbing_retirement_filter(retirement, lagged_retirement):
     },
     "n_periods": 20,
 }
+
+PHELPS_DEATON_NO_BORROWING = {
+    "functions": {
+        "utility": phelps_deaton_utility_with_filter,
+        "next_wealth": next_wealth,
+        "next_wealth_constraint": next_wealth_constraint,
+        "consumption_constraint": consumption_constraint,
```

Comment:
Are we sure we need both constraints here?

Reply:
No, this is redundant.